src/index.js

```diff
@@ -37,7 +38,14 @@ bot.on('ready', async () => {
 });
 
 bot.on('message',  async message => {
-
+    //loop for filter command, is essentially always on but by default the array SHOULD BE EMPTY
```

Comment:
You could use this alternatively: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Would just be an if statement like: 
if(config.FILTER_LIST.some(word => message.content.toLowerCase().includes(word))){
    message.delete()
  }})